### PR TITLE
#167749948 Deploy base application to Heroku

### PR DESCRIPTION
**Pull request by daviddamilola**
#### What does this PR do?
 Deploys barefoot nomad base application to Heroku
#### Description of Task to be completed?
* Add heroku.yml 
* Add app.json
#### How should this be manually tested?
* Access the repo from https://magma-backend-staging.herokuapp.com/
#### Any background context you want to provide?
* To deploy develop branch, the branch with the start script will have to be merged.
* To enable app reviews this branch will have to be merged with the develop branch, else Heroku will add a commit to develop.  
#### What are the relevant pivotal tracker stories?
[#167749948](https://www.pivotaltracker.com/story/show/167749948)

### CODE REVIEW

**Review by hustlaviola — CHANGES_REQUESTED** [review]
Good job on this PR. You can change the *Relevant PT story* to display just the story id by following this format
`[story_id](https://www.pivotaltracker.com/story/show/story_id)`

**Comment by daviddamilola** [comment]
> Good job on this PR. You can change the _Relevant PT story_ to display just the story id by following this format
> [story_id](https://www.pivotaltracker.com/story/show/story_id)

changes have been implemented.

**Review by davidteejay — CHANGES_REQUESTED** [review]
In the `app.json`, are you not supposed to put environment variables inside the `env` object?

**Review by davidteejay — COMMENTED** [review]
And also, you need to change the title. You can look at the other PRs for reference

**Review by davidteejay — APPROVED** [review]

**Review by davidteejay — APPROVED** [review]

**Review by Naimaoye — COMMENTED** [review]
good job. I think the parts that has NIL in your comment should be removed since there is nothing to show

**Comment by daviddamilola** [comment]
> good job. I think the parts that have NILL in your comment should be removed since there is nothing to show

The changes have been made

File: Dockerfile [hunk 1]
```diff
@@ -8,4 +8,4 @@ RUN npm install
 COPY --chown=node:node . .
 
 EXPOSE 5000
-CMD [ "node", "./src/index.js" ]
+CMD [ "node", "./build/index.js" ]
```
Comment (ascii-dev) [inline Dockerfile]: Kindly add a new line at the end of this file. Thanks

Comment (daviddamilola) [reply Dockerfile]: A new Line has been added.

> Kindly add a new line at the end of this file. Thanks

**Review by hustlaviola — APPROVED** [review]
Good job

**Review by davidteejay — APPROVED** [review]

**Review by ibraheem-adeyemo — APPROVED** [review]

**Review by Naimaoye — APPROVED** [review]

**Review by ldonjibson — APPROVED** [review]

**Review by ascii-dev — APPROVED** [review]
LGTM

**Pull request merged**